### PR TITLE
Set cookie domain

### PR DESCRIPTION
**Pull request by ryb73**

### CODE REVIEW

File: bsconfig.json [hunk 1]
```diff
@@ -13,5 +13,8 @@
   "bs-dev-dependencies": [
     "@glennsl/bs-json"
   ],
-  "refmt": 3
-}
\ No newline at end of file
+  "refmt": 3,
+  "warnings": {
+    "number": "-105"
```
Comment (ryb73) [inline bsconfig.json:18]: Unrelated to the cookie update; ignore new bucklescript warning "BuckleScript warning: redirect : the external name is inferred from val name is unsafe from refactoring when changing value name"

**Comment by ncthbrt** [comment]
Looks straightforward enough.
@ryb73 You tested this?

**Comment by ryb73** [comment]
Yep, it worked for me. I could add a unit test but the existing test suite
didn't seem to be working for me.

**Comment by ncthbrt** [comment]
Ok. I'll merge. 
Btw, the test suit requires that the test server is running, it may be why you're having difficulty.

**Pull request merged**